data/persons.txt

```diff
@@ -1850,3 +1849,149 @@ ship "Modified Ladybug"
+	phrase
+		word
+#		Marksman
+			"If you're interested in helping out but don't have time, consider being a proofreader! You can take my job, and I get to laze around doing nothing!"
+			"If you see something, and it shoots at you, cheat your way to victory!"
+			"The number of times someone breaks the fourth wall, it's mind-boggling!"
+			"Look, we aren't here for your money. We do all this for free and look where that gets us."
+			"If you're interested in helping out but don't have time, consider looking through the proposals on GitHub! Sometimes, a new perspective is needed!"
+			"What do you mean you don't have a save file for this pull request!?"
+			"This statement is, intentionally wrote wrongly to make point."
+
+#		Roadrunner
+			"Typo detected. Initiating weapons engagement."
+			"Code non-functional. Destroy them."
+
+#		Saugia
+			"Yea... way too much detail. I can't see the cockpit anymore."
+			"Just don't go ham on details."
+			"These areas are looking much better, though. The contrast of the lesser details here and the more elsewhere is a lot nicer."
+			"I'm just trying to see this half that size and I'm losing a lot of detail."
+			"It's awesome, but far too detailed."
+			"Now there's a lot less detail conflicting with one another."
+
+#		tibetiroka
+			"It's just the Steam CI, no need to panic."
+			"Did you check for nullptr?"
+			"		How do we indent after line breaks again?"
+			"Please put the includes in alphabetical order."
+			"Always keep three empty lines between functions."
+			"Why do Aliens speak English?"
+
+#		1010todd
+			"Style check."
+			"I'll give this a better look later."
+			"What is this for?"
+			"This should be a private member."
+			"Comments?"
+			"Please remove unnecessary space."
+
+			"Looks good to me."
+			"Is this the best name?"
+			"The standard copyright header was changed recently."
+
+			"Please use the template."
+			"The decision to implement fancy quote is one of the worst in computer history."
+			"	This needs another indent."
+#		EjoThims
+	phrase
+		word
+			"If you want your pull request to not be merged, "
+			"If you prefer your pull request never be merged, "
+			"If you prefer your pull request sits idle, "
+			"If you want your pull request to be closed, "
+		word
+			"make sure you keep adding content to it"
+			"always change unrelated things"
+			"do not ever test it and don't add integration tests. Whatever those even are, they're apparently important"
+			"try your best to avoid respecting existing syntax and conventions"
+			"just forget about it for a while"
+			"use the forbidden word: 'realism'"
+			"bury the new feature in semi-relevant data changes"
+			"do not, under any circumstances, ever be willing to compromise"
+			"make sure to completely ignore what the reviewers tell you to do"
+			"skip over writing an issue about it first"
+		word
+			"!"
+			"."
+			". Trust me! It works."
+			". You shouldn't do those things, though! I'm just kidding."
+
```

Comment:
yo I got a couple.

```suggestion

#		bene_dictator
	phrase
		word
			"Just a syntax review. Hold on, I gotta rewrite this quickly. Can you delay the merge?"
			"People who complain about HR don't actually appreciate what we folks do. The number of pull requests that went into that baby..."
			"I was requested for spelling? Oh my goodness, this needs a rewrite..."
			"One more review. Pretty please?"
			"Just one more review before merging."
			"Capitalise that proper noun. Use American English for consistency. For goodness' sake, that needs a comma!"
			"Comma here. Comma doesn't go there. I think that one needs a comma? Hold on, what was a comma again?"
			"Unidentified sentence appears to be lacking comma. Very dangerous, highly likely to incite multiple suggestions on where comma should go. Calling backup reviewers."
			"Commas are your greatest friends."
			"Commas are your greatest enemies."

```
I wonder if mentioning HR is too sensitive... but it's all in good fun. Saying capitalise instead of capitalise is intentional.

(Did this on my ipad so hopefully those are real indents...)

Reply:
I think the budget currently is 4-8 quotes per reviewer, so the HR one is probably gonna burn.